core/src/main/scala/org/ensime/indexer/domain.scala

```diff
@@ -127,6 +130,46 @@ final case class MethodName(
   def fqnString = owner.fqnString + "." + name + descriptor.descriptorString
 }
 
+/**
+ * Generics signature
```

Comment:
if you want a comment delimiter to separate sections in the file, the convention thus far has been 80 comment characters of `//`